src/commands/user-commands/tutor-log.ts

```diff
@@ -0,0 +1,103 @@
+import { Command } from "../../interfaces/Command";
+import { commandLog, sendError } from "../../utils/logs";
+import { SlashCommandBuilder } from "discord.js";
+import { reportOptions } from "../../utils/options";
+import { TutorLogInsert } from "../../utils/types";
+import { getContactId, getTutorId, insertTutorLog } from "../../utils/supabase";
+import { createEmbeded } from "../../utils/embeded";
+
+
+export const tutorlog: Command = {
+  
+    data : new  SlashCommandBuilder()
+      .setName("tutor-log")
+      .setDescription("Log your tutor hours")
+      .addStringOption((option) =>
+      option
+      .setName("tutoring-type")
+      .setDescription("Online or in-person?")
+      .addChoices(
+        {name: 'In Person', value: 't-ip'},
+        {name: 'Online Voice Chat', value: 't-ov'},
+        {name: 'Online Text Chat', value: 't-ot'}
+        )
```

Comment:
We want to add these options dynamically from the db by creating a new `supabase.ts` function to fetch the active tutoring types as well as extracting this logic into a function in `options.ts`. See line 40 in the `grant-membership.ts` command as well as the `membershipCodeOptions` function it uses from `options.ts`.